backend/users/utils.py

```diff
@@ -1,5 +1,156 @@
 import uuid
 
+## Define constants
+
+# Language names to code (Google Translate)
+LANG_NAME_TO_CODE_GOOGLE = {
+    "English": "en",
+    "Assamese": "as",
+    "Bhojpuri": "bho",
+    "Bengali": "bn",
+    "Bodo": "brx",
+    "Dogri": "doi",
+    "Dhivehi": "dv",
+    "Konkani": "gom",
+    "Gujarati": "gu",
+    "Hindi": "hi",
+    "Kannada": "kn",
+    "Kashmiri": "ks",
+    "Mizo": "lus",
+    "Maithili": "mai",
+    "Malayalam": "ml",
+    "Manipuri": "mni-Mtei",
+    "Marathi": "mr",
+    "Nepali": "ne",
+    "Odia": "or",
+    "Punjabi": "pa",
+    "Sanskrit": "sa",
+    "Santali": "sat",
+    "Sindhi": "sd",
+    "Sinhala": "si",
+    "Tamil": "ta",
+    "Telugu": "te",
+    "Urdu": "ur",
+}
+
+# Language code to language name GOOGLE TRANSLATE
+LANG_CODE_TO_NAME_GOOGLE = {
```

Comment:
You can just do the following to avoid duplicate code:

```py
LANG_CODE_TO_NAME_GOOGLE = {
  lang_code: lang_name
  for lang_name, lang_code in LANG_NAME_TO_CODE_GOOGLE.items()
}
```

Reply:
Similarly for `LANG_NAME_TO_CODE_ULCA`

Reply:
Okay, I will do this. This is a helpful trick.